pkg/ocm/list.go

```diff
@@ -0,0 +1,41 @@
+package ocm
```

Comment:
Can you rename this file policylist.go? I'm expecting there to be more structs added to this pkg